portfolio/src/main/webapp/index.html

```diff
@@ -25,12 +23,12 @@ <h1 class="header-item">Ethan Mooney</h1>
           <div class="about-container">
               <div class="about-item-left">
                 <div class="about-item-left-item">
-                     <img src="/images/ethan-p1.jpeg" width="100px">
-                     <img src="/images/ethan-p2.jpeg" width="100px">
-                     <img src="/images/ethan-p3.jpeg" width="100px">
-                     <img src="/images/ethan-p4.jpeg" width="100px">
-                     <img src="/images/ethan-p5.jpeg" width="100px">
-                     <img src="/images/ethan-p6.JPG" width="100px">
+                     <img src="/images/ethan-p1.jpeg" alt="Ethan on bridge" width="100px">
+                     <img src="/images/ethan-p2.jpeg" alt="Ethan on cliff side" width="100px">
+                     <img src="/images/ethan-p3.jpeg" alt="Ethan by Google sign" width="100px">
+                     <img src="/images/ethan-p4.jpeg" alt="Ethan and CSSI friends" width="100px">
+                     <img src="/images/ethan-p5.jpeg" alt="Ethan in wheat field" width="100px">
+                     <img src="/images/ethan-p6.JPG" alt="Ethan CSSI graduation" width="100px">
```

Comment:
nit: This file extension is inconsistent with the others.

Reply:
Yeah idk why it did that, all came from the same place, can change.